conf/init/nginx_conf_create.py

```diff
@@ -108,6 +109,7 @@ def generate_server_config(config):
         maximum_cnr_layer_size=maximum_cnr_layer_size,
         enable_rate_limits=enable_rate_limits,
         static_dir=STATIC_DIR,
+		max_upload_size=max_upload_size,
```

Comment:
Fix indentation

Reply:
@ibazulic It looks like the indentation is still off. Let me know if you need help pushing your local changes, if it looks different on your filesystem.